ledger/Cargo.toml

```
@@ -82,13 +82,7 @@ thiserror = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
 tokio-stream = { workspace = true }
 trees = { workspace = true }
-
-[dependencies.rocksdb]
-# Avoid the vendored bzip2 within rocksdb-sys that can cause linker conflicts
-# when also using the bzip2 crate
-version = "0.23.0"
-default-features = false
-features = ["lz4"]
+rocksdb= { workspace = true, features = ["lz4"] }
```

```suggestion
rocksdb = { workspace = true, features = ["lz4"] }
```

also, please keep the comment here.